test/helpers/field_enums.py

```diff
@@ -90,6 +90,24 @@ def select_query(cls, table="c_w20_transfers"):
         return super().select_query(table)
 
 
+class CW20BalanceChangeFields(NamedFields):
+    id = 0
+    balance_offset = 1
+    contract = 2
+    account_id = 3
+    event_id = 4
+    transaction_id = 5
+    block_id = 6
+
+    @classmethod
+    def select_query(cls, table="c_w20_balance_changes"):
+        return super().select_query(table)
+
+    @classmethod
+    def by_execute_contract_method(cls, method):
+        return f"SELECT c_w20_balance_changes.id, balance_offset, c_w20_balance_changes.contract, account_id FROM c_w20_balance_changes, execute_contract_messages WHERE execute_contract_messages.id = execute_contract_message_id and method = '{method}'"
```

Comment:
It looks like you had to write this out (as opposed to using `cls.select_query`) because you needed to disambiguate the `id` column. I think we could add a boolean argument to `NamedFields.select_query` called `prefix` or something which will append the table name and the dot to the column names. Then we should be able to reduce this further.